packages/cli/src/commands/experimental/templates/sentryWeb.ts.template

```diff
@@ -1,5 +1,5 @@
+import { BrowserTracing } from '@sentry/browser'
```

Comment:
I think we can get rid of this, and instead just do `Sentry.BrowserTracing`, using `Sentry` from the `'@sentry/react'` import below. It's what they do here https://docs.sentry.io/platforms/javascript/guides/react/#configure

How does that sound to you @ageddesi?

Reply:
Yer sounds good, looks like I left that in by mistake :P